ee/modules/381-ltwo-load-balancer/templates/monitoring.yaml

```diff
@@ -0,0 +1,2 @@
+{{- include "helm_lib_grafana_dashboard_definitions" . }}
```

Comment:
TODO (not now, to think):
* make a module metallb-common, which is enabled if one of modules is enabled (mlb or l2lb).
* this modele deploys grafana dashboars and alerts and whatever.